File: app/src/main/java/com/alvinhkh/buseta/nlb/model/NlbEtaRes.java
package com.alvinhkh.buseta.nlb.model;

import com.google.gson.annotations.SerializedName;

public class NlbEtaRes {

    @SerializedName("estimatedArrivalTime")
    public ETA estimatedArrivalTime;

    public class ETA {

        @SerializedName("html")
        public String html;

        public String toString() {
            return "{html=" + html + "}";
        }

    }

    public String toString() {
        return "NlbEtaRes{estimatedArrivalTime=" + estimatedArrivalTime + "}";
    }

}

File: app/src/main/java/com/alvinhkh/buseta/nlb/model/NlbEtaRes.kt
package com.alvinhkh.buseta.nlb.model

import com.google.gson.annotations.SerializedName

data class NlbEtaRes(
        @SerializedName("estimatedArrivalTime")
        var estimatedArrivalTime: ETA = ETA()
) {
    data class ETA(
            @SerializedName("html")
            var html: String = ""
    )
}
